pybliometrics/scopus/utils/startup.py

```diff
@@ -1,21 +1,72 @@
-import configparser
+import os
+from configparser import ConfigParser
 from collections import deque
 from pathlib import Path
+from typing import List, Dict, Optional, Type
+from warnings import warn
 
 from pybliometrics.scopus.utils.constants import CONFIG_FILE, RATELIMITS
 from pybliometrics.scopus.utils.create_config import create_config
 
-# Read/create config file (with fixture for RTFD.io)
-config = configparser.ConfigParser()
-config.optionxform = str
-try:
-    if not CONFIG_FILE.exists():
-        config = create_config()
-    else:
-        config.read(CONFIG_FILE)
-    KEYS = [k.strip() for k in config.get('Authentication', 'APIKey').split(",")]
-except EOFError:
-    pass
+CUSTOM_DIR = None
+CUSTOM_KEYS = None
 
-# Throttling params
 _throttling_params = {k: deque(maxlen=v) for k, v in RATELIMITS.items()}
+
+def get_config() -> Type[ConfigParser]:
+    """Auxiliary function to get the config parser"""
+    # Read/create config file (with fixture for RTFD.io)
+    config = ConfigParser()
+    config.optionxform = str
+
+    if not CUSTOM_DIR and not CONFIG_FILE.exists():
+        warn('Please create a configuration file by initializing Pybliometrics with init().\n'
+             'For more information visit: '
+             'https://pybliometrics.readthedocs.io/en/stable/configuration.html')
+    else:
+        if CUSTOM_DIR:
+            config.read(CUSTOM_DIR)
+        else:
+            warn('Please initialize Pybliometrics with init().\n'
+                 'For more information visit: '
+                 'https://pybliometrics.readthedocs.io/en/stable/configuration.html', FutureWarning)
+            config.read(CONFIG_FILE)
+
+    return config
+
+def get_config_path() -> Type[Path]:
+    """Auxiliary function to get the configuration file path"""
+    return CUSTOM_DIR or CONFIG_FILE
+
+def get_keys() -> List[str]:
+    """Auxiliary function to get the API keys"""
+    if CUSTOM_KEYS:
+        keys = CUSTOM_KEYS
+    else:
+        config = get_config()
+        keys = [k.strip() for k in config.get('Authentication', 'APIKey').split(",")]
+    return keys
+
+def init(config_dir: Optional[str] = CONFIG_FILE, keys: Optional[List[str]] = None) -> None:
```

Comment:
I actually thought somehow that the init() function would not just create the configuration, but also read it, and then pass on the values. Is this not possible?